src/main.cc

```diff
@@ -1203,6 +1203,8 @@ mainInitialize(void)
     FwdState::initModule();
     SBufStatsAction::RegisterWithCacheManager();
 
+    AsyncJob::RegisterWithCacheManager();
```

Comment:
We are trying to reduce the number of explicit dependencies in main.cc (see commit 230d441). However, today, main.cc already knows about the AsyncJob class, and replacing this call with CallRunnerRegistrator() would require adding a few extra code lines because AsyncJob.cc does not use a RegisteredRunner yet. Should we do this refactoring in this PR?